docs/content/tools/xdebug.md

```diff
@@ -4,210 +4,182 @@ aliases:
+`PHP_IDE_CONFIG` variable. This can be done using the following commands:
 
-- Right mouse click on project name, then "Properties" from the dropdown menu
-- Under "Sources", set correct Web Root folder by clicking "Browse" button (usually it's `docroot`)
-- Under "Run Configuration", use project's virtual host (`VIRTUAL_HOST`) to set the Project URL
-- Click "OK" to save project properties
-
-5) Set a breakpoint wherever you like
-6) In NetBeans, with the whole project selected or one of the project files opened and active, press `<CTRL> + <F5>` on your keyboard to start the debugger
-
-## Debugging with Atom
-
-1) Follow the Setup instructions to enable Xdebug in cli
-2) Install [PHP-Debug Plugin](https://atom.io/packages/php-debug)
-3) Configure PHP Debug Settings (Preferences > Packages)
-
-- Search for "php-debug"
-- Click "Settings" button below plugin
-- "Server" can be set to `*`
-- "Server Listen Port" should be set to 9000
-- Make sure "Continue to listen for debug sessions even if the debugger windows are all closed" is checked. This will make the debugger window open automatically.
-
-## Debugging with Visual Studio Code
-
-1) Install [PHP_Debug Plugin](https://marketplace.visualstudio.com/items?itemName=felixfbecker.php-debug)
-2) Configure PHP Debug Settings
-
-- Click Debug Icon in left sidebar
-- Click Configuration Gear in top bar
-- Set port to 9000
-- Add Path Mappings should be configured properly
-
-
-Here is an an example of what `launch.json` should look like:
-```json
-{
-  "version": "0.2.0",
-  "configurations": [
-    {
-      "name": "Listen for XDebug",
-      "type": "php",
-      "request": "launch",
-      "port": 9000,
-      "pathMappings": {
-        "/var/www/": "${workspaceRoot}"
-      }
-    },
-    {
-      "name": "Launch currently open script",
-      "type": "php",
-      "request": "launch",
-      "program": "${file}",
-      "cwd": "${fileDirname}",
-      "port": 9000
-    }
-  ]
-}
+```bash
+fin config set --env=local 'PHP_IDE_CONFIG=serverName=${VIRTUAL_HOST}'
+fin project start
 ```
 
-![Visual Studio configuration](/images/xdebug-vscode2.png)
+{{% notice warning %}}
+The script you are trying to debug must reside within the project root directory (`/var/www/`on the server) or 
+PHPStorm won't be able to access the scripts's source code and debug it.
+{{% /notice %}}
 
-For debugging Drush commands in VSCode with XDebug and Docksal, you will need to add a path mapping to the Drush executable.
+### CLI Scripts: Drush {#phpstorm-drush}
 
-### File Mapping Drush 8.x
+To debug custom Drush commands, make the following additional adjustments in PHPStorm settings:
 
-- Add `"/usr/local/bin/drush": "${workspaceRoot}/bin/drush"` to `pathMappings` in your `launch.json` file.
+1. Go to **Preferences > Languages & Frameworks > PHP > Debug**
+2. Uncheck **Force break at the first line when no path mapping specified**
```

Comment:
```suggestion
1. Uncheck **Force break at the first line when no path mapping specified**
```

Reply:
I prefer real numbers rather than using `1.` for every line.

Reply:
No probs. It's not critical, but using `1.` everywhere is considered best practice, because it allows changing lists later without changing every line.